platform/src/common/components/Calendar/CustomCalendar.jsx

```
@@ -18,56 +15,63 @@
- * @param {String} props.position
- * @param {String} props.className
- * @returns {JSX.Element}
- * @description CustomCalendar component
+ * Helper function to check if a value is a valid Date object.
+ *
+ * @param {any} date - The value to check.
+ * @returns {boolean} - True if valid Date, else false.
  */
-const CustomCalendar = ({ initialStartDate, initialEndDate, className }) => {
+const isValidDate = (date) => {
+  return date instanceof Date && !isNaN(date);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)